services/src/test/java/org/exoplatform/task/rest/TestTaskRestService.java

```diff
@@ -153,9 +153,17 @@ public void testGetTasksByProjectId() throws Exception {
     TaskQuery taskQuery = new TaskQuery();
     List<Long> allProjectIds = new ArrayList<Long>();
     allProjectIds.add((long) 1);
+    ProjectDto project1 = new ProjectDto();
+    project1.setName("project1");
+    Set<String> manager = new HashSet<String>();
+    manager.add("root");
+    project1.setManager(manager);
+    when(projectService.getProject(project1.getId())).thenReturn(project1);
+
     when(taskService.findTasks(taskQuery,0,-1)).thenReturn(tasks);
+
     // When
-    Response response = taskRestService.getTasksByProjectId((long)1,0,-1,false,false, false);
+    Response response = taskRestService.getTasksByProjectId(project1.getId(),0,-1,false,false, false);
```

Comment:
We need to test the failure use case (returning Response.Status.UNAUTHORIZED)
In fact, we want to ensure that future maintenance on this file doesn't break the newly implemented rule: `Users shouldn't get access only to project where they are member`. We will have to test the success and failure use cases (OK and UNAUTHORIZED).